react/Header/Header.js

```diff
@@ -16,17 +16,22 @@ const employerLinkHref = locale => locale === 'NZ' ?
   'https://talent.seek.co.nz/' :
   'https://talent.seek.com.au/';
 
+const AUTHENTICATED = 'authenticated';
```

Comment:
I'm not sure it's worth the effort, given we're using `propTypes`, and the API would be a bit awkward. The best I can come up with as a scalable pattern would look something like this, which I'm not a fan of:

```js
import { Header } from 'seek-style-guide/react';
const { AUTHENTICATED, UNAUTHENTICATED, PENDING } = Header.authenticatonStatus;
```